3561/0/about_dice_project.rb

```diff
@@ -0,0 +1,66 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+class DiceSet
+  attr_reader :values
+
+  def roll(count)
+    @values = Array.new(count).collect! { |_item| Random.new.rand(1..6) }
```

Comment:
O_o
Why it can't be something like that `Array.new(count) { rand(1..6) }`
And you can make it even better! (without `{}` for example)